internal/wasi/wasi.go

```diff
@@ -1383,87 +1377,69 @@ func (a *wasiAPI) PathLink(ctx wasm.Module, oldFd, oldFlags, oldPath, oldPathLen
 	return wasi.ErrnoNosys // stubbed for GrainLang per #271
 }
 
-const (
-	// WASI open flags
-	oflagCreate = 1 << iota
-	// TODO: oflagDir
-	oflagExclusive
-	oflagTrunc
-
-	// WASI FS rights
-	rightFDRead  = 1 << iota
-	rightFDWrite = 0x200
-)
-
-func posixOpenFlags(oFlags uint32, fsRights uint64) (pFlags int) {
```

Comment:
#390 re-introduce some aspects of this later. it would ignore fsRights as they aren't helpful or likely to be implemented in any consistent way. Rather, pay attention to mount config (static) oFlags and dirFlags See